tests/installation/upgrade/remove_previous_repo.pm

```diff
@@ -0,0 +1,19 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles page for detection of previously used repositories.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+
+sub run {
+    my $previous_repo = $testapi::distri->get_previous_repo();
+    $previous_repo->get_previous_repo();
```

Comment:
Only an assertion here, selecting the row would toggle "Remove" to "Keep" otherwise.